src/components/header/header.tsx

```diff
@@ -1,37 +1,67 @@
-import { persistor } from "../../app/store/store";
 import useAuthentication from "../../hooks/useAuthentication";
-import "./header.css";
 import PlusIcon from "../../assets/plus-icon.webp";
-import Logo from "../../assets/logo.png";
+import LogoCeci from "../../assets/logoCeci.png";
+import LogoutIcon from "../../assets/logout.png"
+import PerfilPicture from "../../assets/perfil.jpg"
+import {MisAlumnos, NuevoAlumno, MisPlanes, Login} from "../../routes/routes"
+import { MainContainer, 
+        SecondaryContainer, 
+        TertiaryContainer , 
+        Logo,  
+        NewStudent, 
+        Logout, 
+        Photo, 
+        Options} from "./header.styles";
+import { useNavigate } from "react-router";
+import { ConfirmDialog } from "../confirm_dialog/ConfirmDialog";
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { clearUser } from "../../app/slices/AuthSlice";
+
 function Header() {
+
+  const dispatch = useDispatch();
   const { isAuthenticated } = useAuthentication();
 
   const handleLogout = () => {
-    persistor.purge();
+    setShowConfirm(false);
+    dispatch(clearUser())
+    navigate(Login);
   };
 
+
+  const navigate = useNavigate();
+
+  const [showConfirm, setShowConfirm] = useState(false);
+
   return (
-    <header className="header">
-      {isAuthenticated && (
-        <>
-          <div className="header-left">
-            <a href="/home">Listado de alumnos</a>
-            <a href="/cuotas">Cuotas mensuales</a>
-          </div>
-          <div className="header-right">
-            <a href="/nuevo-alumno" className="new-student">
-              Nuevo alumno
-              <img src={PlusIcon} alt="Plus" className="plus-icon" />
-            </a>
-            <a href="/login" onClick={handleLogout}>
-              Salir
-            </a>
-            <img src={Logo} alt="Logo" className="logo"></img>
-          </div>
-        </>
+    isAuthenticated && (
+    <MainContainer>
+      <SecondaryContainer>
+        <Logo><img src={LogoCeci} alt="Logo" /></Logo>
+        <Options>Calendario</Options>
```

Comment:
Ojo con el nombre del styled component acá, cada elemento `<Options>` no son múltiples opciones sino que es una sola, por lo que deberia ir en singular

```suggestion
        <Option>Calendario</Option>
```